### ISSUE TITLE
bug: No default tab after login on OpenEMR 7.0.2 (upgrade from 7.0.2)

### ISSUE THREAD

**Issue by stephenwaite** [openemr/openemr#7040]
### Describe the bug

[concise description of what the bug is.](https://community.open-emr.org/t/no-default-tab-after-login-on-openemr-7-0-2-upgrade-from-7-0-2/21527?u=stephenwaite)

### To Reproduce

1. Add a '....'

### Client configuration

- Browser:
- OpenEMR version:
- Operating system:

**Comment by stephenwaite** [comment]
looks like we lost the ability for users to have their own default tabs

**Comment by bradymiller** [comment]
yep, I must of missed that on the review for that feature (where migrated that setting to lists to allow more flexibility (ie. added ability for not just 2 tabs (could be any number) and added ability to add more tabs (ie. custom tabs). Would be very nice to figure out mechanism to support per user setting on this!

**Comment by bradymiller** [comment]
Would  consider something like.
1. A new tab in the user settings that lists each of the items from the list with a toggle (and works the same way as the other settings in user settings where it shows what is the default setting and ability to reset to default).
2. Then incorporate these user settings when bringing up the tabs after login

**Comment by stephenwaite** [comment]
hi @bradymiller, would that mean adding it to the globals metadata array or would you find a new way to make it user specific without touching globals?
